.github/workflows/ai_robots_update.yml

```diff
@@ -0,0 +1,34 @@
+name: Updates for AI robots files
+on:
+  push:
+    branches:
+    - "main"
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  dark-visitors:
+    runs-on: ubuntu-latest
+    name: dark-visitors
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+      - run: |
+          pip install beautifulsoup4 requests
+          git config --global user.name "dark-visitors"
+          git config --global user.email "dark-visitors@users.noreply.github.com"
```

Comment:
Why are these configs made here? And should they be changed for the conversion step?

Reply:
I don't know (x2).

Reply:
Actually, the purpose of these configs is to make the changes produced by the dark visitors website look as if they were produced by a dark-visitors user. I'm not sure this is entirely necessary. The commit message should be sufficient on its own.